projects/igniteui-angular/src/lib/grids/filtering/excel-style/excel-style-column-moving.component.ts

```diff
@@ -51,6 +52,10 @@ export class IgxExcelStyleColumnMovingComponent {
         this.grid.moveColumn(this.column, targetColumn, moveDirection);
     }
 
+    private get visibleColumns() {
```

Comment:
You could move this property up in the file and place it where the other properties are.

Reply:
done